.evergreen/.evg.yml

```diff
@@ -480,18 +480,24 @@ axes:
       - id: "rhel70"
         display_name: "RHEL 7.0"
         run_on: rhel70-small
-  - id: "special-os"
-    display_name: OS
-    values:
-      - id: "rhelzseries"
-        display_name: "RHEL 7.2 Z-Series"
-        run_on: rhel72-zseries-test
-      - id: "rhelpower"
-        display_name: "RHEL 7.1 PowerPC"
-        run_on: rhel71-power8-test
-      - id: "ubuntu1604-arm64-small"
-        display_name: "Ubuntu 16.04 ARM64"
-        run_on: ubuntu1604-arm64-small
+  #- id: "special-os"
```

Comment:
To make sure I'm understanding correct, the intention is that we'll add these back once the build team makes the changes we require? (Ditto for the other commented out sections)

Reply:
Awesome, thanks